### PR TITLE
bpo-39200: Correct the error message for min/max builtin function

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue39200

### CODE REVIEW

**Comment by tirkarthi** [comment]
Seems like a duplicate of #17813

**Comment by tirkarthi** [comment]
Sorry, misread the diff and PR linked to issue as duplicate solution.

**Comment by corona10** [comment]
@tirkarthi 
> Sorry, misread the diff and PR linked to issue as duplicate solution. 

Yes, IMHO, The issue title should be updated as I wrote at the bpo-39200

**Comment by pablogsal** [comment]
Let's hold until we decide how to go in https://github.com/python/cpython/pull/17813

File: Python/bltinmodule.c [hunk 1]
```diff
@@ -1591,8 +1591,12 @@ min_max(PyObject *args, PyObject *kwds, int op)
 
     if (positional)
```
Comment (pablogsal) [inline Python/bltinmodule.c]: Now that we are modifying this code, add braces to the `if (positional)` per PEP7

**Review by pablogsal — APPROVED** [review]
LGTM

**Comment by corona10** [comment]
@pablogsal Updated :)

**Review by pablogsal — APPROVED** [review]

**Pull request merged**